datastream-common/src/main/java/com/linkedin/datastream/metrics/BrooklinMetrics.java

```diff
@@ -5,19 +5,11 @@
  */
 package com.linkedin.datastream.metrics;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.concurrent.atomic.AtomicInteger;
-
-
 /**
  * Base class for metric categories with support for metric deregistration.
- * It uses reference counting for deregistering aggregate metrics which can
- * only happen when all keyed metrics of the same name have been deregistered.
+ * Underlying DynamicMetricsManager will take care of ref-count.
```

Comment:
nit:  reword: Underlying DynamicMetricsManager will take care of maintaining ref-counts.